modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -6663,7 +6663,8 @@ export const console: ConsoleNS = {
                     columns: {
                         actions: "Actions",
                         lastModified: "Modified Time",
-                        name: "Group"
+                        name: "Group",
+                        source: "User-store"
```

Comment:
Let's make this term consistent with other places.

Reply:
This term was already changed to "User store" in a following PR.
https://github.com/wso2/identity-apps/blob/master/modules/i18n/src/translations/en-US/portals/console.ts#L6721